torch/_torch_docs.py

```diff
@@ -2009,6 +2009,13 @@ def merge_dicts(*dicts):
 Returns a new tensor with the hyperbolic cosine  of the elements of
 :attr:`input`.
 
+.. note::
+    Depending on the size and contiguity of the input, the implementation
```

Comment:
When :attr:\`input\` is on the CPU, the implementation of `torch.cosh` may use the Sleef library, which rounds very large results to infinity or negative infinity. See <link> for details.


depending on the size and contiguity of :attr:\`input\`, the implementation of `torch.cosh`